metadata-ingestion/src/datahub/ingestion/source/openapi.py

```diff
@@ -269,7 +297,7 @@ def close(self):
 
 class OpenApiSource(APISource):
     def __init__(self, config: OpenApiConfig, ctx: PipelineContext):
-        super().__init__(config, ctx, "OpenApi")
```

Comment:
1. Is this intentional?
2. what happens to previously ingested urns? when you re-run ingestion they get new urns, so the old entities will continue to exist?

Reply:
Yes, it's intentional. I spotted that there was issues with finding the icon, so I uniformed all to the lowercase format, as in other ingestion plugins.
The older urns will continue to exist as different entities.